packages/formation-react/stories/sidenav.stories.mdx

```diff
@@ -0,0 +1,112 @@
+                    </li>
+                    <li>
+                      <a href="#">Link</a>
+                    </li>
+                    <li>
+                      <a href="#">Link</a>
+                    </li>
+                  </ul>
+                </div>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+      <div class="vads-l-col--12 vads-u-padding-x--2p5 medium-screen:vads-l-col--8 large-screen:vads-l-col--9">
+        <div style={{ opacity: 0.5 }}>
+          <h1 class="vads-u-margin-top--0">Page content would appear here.</h1>
+          <p class="va-introtext"> Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce dictum mauris at laoreet dictum.</p>
+          <p>Mauris ultrices pellentesque lobortis. Ut sed libero vitae orci maximus molestie et id justo. Donec feugiat elit sed pretium condimentum. Sed in cursus dui, eget sodales nisl. In hac habitasse platea dictumst. Fusce eu lacus in purus sodales ullamcorper scelerisque in massa. Duis venenatis, orci nec lacinia dictum, nisl eros lobortis sapien, vel pharetra nisi nulla quis tortor.  </p>
+          <p>Mauris ultrices pellentesque lobortis. Ut sed libero vitae orci maximus molestie et id justo. Donec feugiat elit sed pretium condimentum. Sed in cursus dui, eget sodales nisl. In hac habitasse platea dictumst. Fusce eu lacus in purus sodales ullamcorper scelerisque in massa. Duis venenatis, orci nec lacinia dictum, nisl eros lobortis sapien, vel pharetra nisi nulla quis tortor.  </p>
+          <p>Mauris ultrices pellentesque lobortis. Ut sed libero vitae orci maximus molestie et id justo. Donec feugiat elit sed pretium condimentum. Sed in cursus dui, eget sodales nisl. In hac habitasse platea dictumst. Fusce eu lacus in purus sodales ullamcorper scelerisque in massa. Duis venenatis, orci nec lacinia dictum, nisl eros lobortis sapien, vel pharetra nisi nulla quis tortor.  </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</Canvas>
+
+* All elements, including the `<script>` tag in the snippet are required. An update to Formation's script may address this.
+* The `va-btn-sidebarnav-trigger` may appear anywhere in the DOM, as long as it appears in an expected place on mobile devices.
+
+
+## Accessibility
+* Ensure the side navigational system is keyboard accessible. Users should be able to tab through each link.
+
+## Usability
+
+### When to use
+* To display the “sub-navigation” within a section of the website.
+
+### When to consider something else
+* If the site has fewer than five pages, consider organizing the page without a navigational hierarchy.
+* If your page already has a horizontal and vertical navigation bar, consider ways to simplify your navigation system.
+* If your content is within a frame or sub-area of a page, consider ways to simplify your navigation system.
+
+### Guidance
+* Typically used in a [two-column layout]({{ site.baseurl }}/layout/page-templates#two-columns-content-on-right).
```

Comment:
Same for this link.

Reply:
Oh, this one probably won't. :sweat_smile: Good catch.